spock-specs/SpockTestConfig.groovy

```diff
@@ -1,3 +1,9 @@
 report {
   enabled true
 }
+
+runner {
+  parallel {
+    enabled true
```

Comment:
I wonder how much it impacted the Spock test execution itself?

Reply:
I assumed you already did it, testing your implementation with some "acceptance tests" :).

I was curious about the Travis build, but it seems there is something wrong and it wasn't properly triggered.